sympy/stats/tests/test_continuous_rv.py

```diff
@@ -541,6 +541,25 @@ def test_exponential():
     assert where(X <= 1).set == Interval(0, 1)
 
 
+def test_exgaussian():
+    m = Symbol("m")
+    s = Symbol("s", positive=True)
+    l = Symbol("l", positive=True)
+    z = Symbol("z")
+    X = ExGaussian("x", m, s, l)
+
+    assert density(X)(z) == l*exp(l*(l*s**2 + 2*m - 2*z)/2) *\
+        erfc(sqrt(2)*(l*s**2 + m - z)/(2*s))/2
+
+    assert simplify(cdf(X)(z)) == (erf(sqrt(2)*(l*s**2 + m - z)/(2*s)) -
+    1)*exp(l*(l*s**2 + 2*m - 2*z)/2)/2 - erf(sqrt(2)*(m - z)/(2*s))/2 + S(1)/2
```

Comment:
Broad simplify should be avoided in tests. When simplification is necessary you should identify exactly which steps are necessary (e.g. factor, powsimp etc) rather than calling full simplify (this simplification will become part of the test suite and be run over and over again on Travis and elsewhere).

I think it's better for the test to show what the actual output is though. So I would change it to something like:
```
# Note: actual_output simplifies to expected_output.
# Ideally cdf(X)(z) would return expected_output
expected_output = ...
actual_output = ...
assert cdf(X)(z) == actual output
# assert simplify(actual_output) == expected_output
```

That way the test shows what the output is and the comment helps someone to understand that the output is correct although could be simplified. If the output in future changes then someone can notice and see that

Reply:
@ritesh99rakesh can you remove `simplify`?